tests/test-cases/Group23-VIC-Machine-Service/23-04-VCH-Inspect.robot

```diff
@@ -0,0 +1,149 @@
+# Copyright 2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation     Test 23-04 - VCH Inspect
+Resource          ../../resources/Util.robot
+Resource          ../../resources/Group23-VIC-Machine-Service-Util.robot
+Suite Setup       Setup
+Suite Teardown    Teardown
+
+Default Tags
+
+*** Keywords ***
+Secret Install VIC Appliance With Ops Credentials
+    [Tags]  secret
+    Install VIC Appliance To Test Server  certs=${true}  additional-args=--ops-user=%{TEST_USERNAME} --ops-password=%{TEST_PASSWORD} --debug 1
```

Comment:
this is just setting the ops user to administrator@vsphere.local... not sure that really is what we want to do, also wrapping this entire thing in [tags] secret is going to be a real problem when it comes to debugging.

Reply:
> this is just setting the ops user to administrator@vsphere.local... not sure that really is what we want to do
in order to not duplicate with CLI's tests, here it only checks that the ops user configuration goes into VCH config (inspect output matches up with what we put in install parameter). So it doesn't really matter what ops user we set it to. 

> wrapping this entire thing in [tags] secret is going to be a real problem when it comes to debugging
Since we're setting ops user, I guess we don't want to expose `%{TEST_USERNAME}` and `%{TEST_PASSWORD}` in the log?

Reply:
@jzt is adding an ops user to our CI already, just use that and remove the tags secret from here.